app/assets/stylesheets/ie8.css.scss

```diff
@@ -0,0 +1,16 @@
+/* add exemptions styling - mainly to position FRA code */
+span.exmpt {
+  font-weight: 700;
+  float:right;
+  margin-right: -80px;
+  display: block;
+  zoom: 1;
+  position:relative;
+}
+
+label.exemption {
+  background-color: #ededed;
+  border-color: #ededed;
+  padding-right: 100px;
+  width: 66.6666666667%;
+}
```

Comment:
A bit trivial and not worth burning more then a few minutes on (and not worth introducing any complexity etc), but scsslint reports a few warnings
```
ie8.css.scss:1 Comment: Use `//` comments everywhere
ie8.css.scss:2 QualifyingElement: Avoid qualifying class selectors with an element.
ie8.css.scss:3 PropertySortOrder: Properties should be ordered display, float, font-weight, margin-right, position, zoom
ie8.css.scss:4 SpaceAfterPropertyColon: Colon after property should be followed by one space
ie8.css.scss:8 SpaceAfterPropertyColon: Colon after property should be followed by one space
ie8.css.scss:11 QualifyingElement: Avoid qualifying class selectors with an element.
ie8.css.scss:12 ColorVariable: Color literals like `#ededed` should only be used in variable declarations; they should be referred to via variable everywhere else.
ie8.css.scss:13 ColorVariable: Color literals like `#ededed` should only be used in variable declarations; they should be referred to via variable everywhere else.
```